olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -1376,7 +1376,7 @@ spec:
     - name: "IMAGE_pingsource-mt-adapter__dispatcher"
       image: "registry.redhat.io/openshift-serverless-1/kn-eventing-mtping-rhel8@sha256:c5a5b6bc4fdb861133fd106f324cc4a904c6c6a32cabc6203efc578d8f46bbf4"
     - name: "IMAGE_APISERVER_RA_IMAGE"
-      image: "registry.redhat.io/openshift-serverless-1/kn-eventing-apiserver-receive-adapter-rhel8@sha256:ec3c038d2baf7ff915a2c5ee90c41fb065a9310ccee473f0a39d55de632293e3"
+      image: "registry.redhat.io/openshift-serverless-1/kn-eventing-apiserver-receive-adapter-rhel8@"
```

Comment:
This seems wrong
/hold

Reply:
From: https://github.com/openshift-knative/serverless-operator/actions/runs/12768031745/job/35587572755
```
2025-01-14T12:57:14.8367554Z time="2025-01-14T12:57:14Z" level=fatal msg="Error parsing manifest for image: fetching target platform image selected from image index: reading manifest sha256:e0aea70fb92cafe5ca358f66c755b5599e2cd217ac4d8ab83eda36be13cc7a32 in quay.io/redhat-user-workloads/ocp-serverless-tenant/serverless-operator-135/kn-eventing-apiserver-receive-adapter: received unexpected HTTP status: 502 Bad Gateway"
2025-01-14T12:57:41.0884824Z time="2025-01-14T12:57:41Z" level=fatal msg="fetching blob: received unexpected HTTP status: 502 Bad Gateway"
```

Reply:
fixed with new run https://github.com/openshift-knative/serverless-operator/actions/runs/12768646012/job/35591935517
/unhold